documentation/assettracker/topics/setup-and-configuration.rst

```diff
@@ -33,7 +33,11 @@ Since rack sizes can vary, please refer to the following table for assembly conf
 | 52             | Use 52RU assembly kit                                         |
 +----------------+---------------------------------------------------------------+
 
-.. note:: You can use the Expected Rack Units and Top of Rack Offset (RU) fields on the rack's Properties page (*rack → Information → Properties*) to fine-tune your setup or  accommodate additional assembly configurations and rack sizes, as needed. Please attach master modules and extension modules accordingly.
+.. note:: You can use the Expected Rack Units and Top of Rack Offset (RU) fields on the rack's Properties page (*rack → Information → Properties*) to fine-tune your setup or accommodate additional assembly configurations and rack sizes, as needed. Please attach master modules and extension modules accordingly.
+
+Rack assemblies must be configured with a tool to point them to an applicable Data Collector. The first-generation Rack Assemblies will listen on port 502/TCP. The configuration must be done from a Windows machine running the configuration tool on the same network as the Rack Assemblies.
```

Comment:
Second generation is on 5656/TCP if you want to add it. Also there's no requirement for the tool to be on the same network, it's just TCP it'll route fine.

Reply:
Thanks! I will address

Reply:
Addressed.